test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -197,6 +199,49 @@ def check_bc(existing_schemas):
         )
     return is_bc
 
+def check_fc(existing_schemas):
+    new_schema_dict = load_schemas_to_dict()
+    is_fc = True
+    broken_ops = []
+    # TODO skip allow list for now.
```

Comment:
Can support for the skip list be added to this PR, or are there design issues? Like are you thinking of having a separate BC and FC allow lists?

Reply:
The skip list was for BC before, so I was hesitant to include them here. But now I think about it, it probably makes more sense for those schemas to be considered for both back and forward compatibility. I will allow skip list here then. I think we should maintain one single list for both BC and FC. @gmagogsfm What do you think?